docs/maps/mapPoint-series.md

```diff
@@ -0,0 +1,13 @@
+Map point series
+================
+
+A `mappoint` series is a special form of `scatter` series where the points can be laid out on top of the map using lon and lat coordinates.
+
+<iframe style="width: 100%; height: 520px; border: none;" src="https://highcharts.com/samples/embed/maps/demo/mappoint-latlon" allow="fullscreen"></iframe>
+
+Therefore, it is possible here to place a [marker](https://api.highcharts.com/highmaps/plotOptions.mappoint.marker) on a specific position determined from [mappoint.data](https://api.highcharts.com/highmaps/series.mappoint.data). It may be useful as in the case of the above demo, e.g. when marking cities. We overwrite there the default [x](https://api.highcharts.com/highmaps/series.mappoint.data.x) and [y](https://api.highcharts.com/highmaps/series.mappoint.data.y) coordinates with the values of [lat](https://api.highcharts.com/highmaps/series.mappoint.data.lat) and [lon](https://api.highcharts.com/highmaps/series.mappoint.data.lon).
```

Comment:
This allows us to place...

Additions:
* I think x and y don't have to be mentioned at all here, because they are not convenient. Explaining `lat` and `lon` is enough.
* In addition to mentioning `lat` and `lon`, explain how we can use a [geometry](https://api.highcharts.com/highmaps/series.mappoint.data.geometry). The nice thing about using geometry is that it is directly compliant to GeoJSON feature collections and TopoJSON geometries, so that these can be used directly either as `data` or `mapData`.